uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/model/impl/BaseGridData.java

```diff
@@ -397,7 +397,7 @@ protected Range doSetCell(final int rowIndex,
         //If we're not merged just set the value of a single cell
         if (!isMerged) {
             ((BaseGridRow) rows.get(rowIndex)).setCell(_columnIndex,
-                                                       cellSupplier.apply(Pair.newPair(rowIndex, columnIndex)));
+                                                       cellSupplier.apply(new AbstractMap.SimpleEntry<>(rowIndex, columnIndex)));
```

Comment:
@danielezonca what was the reason for changing Pair with Map.Entry? Just curious...

Reply:
@jsoltes 
The goal was to remove the dependency with `org.uberfire.commons.data.Pair` to isolate grid component and I just decided to use a built-in class instead of create a new bean just to wrap two values. No other specific/technical reason :)

Reply:
@danielezonca ok, I struck me right after I wrote it. I thought it was the javafx.util.Pair, didn't realize the connection to uberfire at first.